src/main/resources/templates/full.html

```diff
@@ -3,6 +3,14 @@
 {{> head }}
 <body>
     <h2>Sorry</h2>
-    <p class="lead">Für diese Veranstaltung sind alle verfügbaren Plätze bereits belegt.</p>
+    <p class="lead">Für diese Veranstaltung sind alle verfügbaren Plätze bereits belegt. Bitte meldet Euch bei uns per <a href="mailto:orga@jug-da.de">E-Mail</a>, ggf. können wir Euch doch noch aufnehmen.</p>
+	<p class="lead">
+		<small>
```

Comment:
Ein `<small>` Tag innerhalb eines Paragraphs mit `lead` Klasse ergibt nicht so richtig Sinn, oder!?

Reply:
wahrscheinlich nicht, ich schaue es mir nochmal in den chrome-dev tools an

Reply:
habe es angepasst